Makefile

```diff
@@ -480,7 +480,7 @@ proto-update-deps:
 ###                                Localnet                                 ###
 ###############################################################################
 
-# Run a 4-node testnet locally
+# Run a 4-node testnet locally via docker compose
```

Comment:
Only change appears to be a comment change? Still want this merged?

Reply:
So sorry. Must have been a confusion while handling git. 
New commit has been made with the change included.